Headers/species_mod.F90

```diff
@@ -122,6 +125,9 @@ MODULE Species_Mod
      ! Default background concentration
      REAL(fp)           :: BackgroundVV     ! Background conc [v/v]
 
+     ! Species units
+     CHARACTER(LEN=80)  :: Units            ! Species units (e.g. v/v, days)
```

Comment:
See my comment on the species_database.yml file. Since the units are variable in GEOS-Chem shouldn't this not be part of the species database? If there are fixed time units for transport tracers maybe we should call this "Lifetime_Units" or something like that. The Units for concentrations (that can change throughout GEOS-Chem) would be in State_Chem%Species(N)%Units

Reply:
Per an internal conversation at one of our GCST meetings, I had proposed we include a new field `Units` in the species object. When units are converted, we could update this string per species and use it for error checking. See also my comment [above](https://github.com/geoschem/geos-chem/pull/1816#discussion_r1218345732).

Reply:
There will be a separate GitHub discussion for tracking concentration units. We might want to do something like label the units as "Fixed_Units" in the species database and only list them if the units are fixed (most species will have dynamic units).